### ISSUE TITLE
URL Params not set on web

### ISSUE THREAD

**Issue by danielsogl** [capacitor-community/http#42]
**Describe the bug**
Params defined in the `params` options are not set

**To Reproduce**

This request will be translated into following url
```ts
    const response = await Capacitor.Plugins.Http.request({
      method: 'GET',
      url: 'https://api.rss2json.com/v1/api.json',
      params: {
        rss_url: environment.npmBlogEndpoint,
        api_key: environment.rss2jsonApiKey,
        count: '100'
      }
    });
```

```bash
// expected URL
https://api.rss2json.com/v1/api.json?rss_url=http://blog.npmjs.org/rss&api_key=vaamv0bymd6t8b2lorlzway3ph5wg6gpauwfuk7a&count=100

// Requested URL
https://api.rss2json.com/v1/api.json
```

**Expected behavior**
The url params should been set correctly

**Comment by gbbarroso** [comment]
Running into this as well

**Comment by andutoptal** [comment]
Any ETA on when will the fix for this be released?

**Comment by thomasvidas** [comment]
The current plan is the release a 0.3.0 with this and other fixes released later this week. If you need it immediately, you can clone the repo and run the following to build the project in a `/dist` folder
```bash
npm install
npm run build
```

Closing this as the fix is already fixed in #46 and merged into master
